packages/clerk-js/rspack.config.js

```diff
@@ -96,6 +96,11 @@ const common = ({ mode, variant, disableRHC = false }) => {
             name: 'zxcvbn-common',
             chunks: 'all',
           },
+          baseAccountSDKVendor: {
+            test: /[\\/]node_modules[\\/](@base-org\/account|@noble\/curves|abitype|ox|preact|eventemitter3|viem|zustand)[\\/]/,
```

Comment:
This is needed to avoid bundling `@base-org/account` transitive dependencies in our main vendor chunks